x-pack/plugins/observability/server/ui_settings.ts

```diff
@@ -374,6 +375,15 @@ export const uiSettings: Record<string, UiSettings> = {
     schema: schema.boolean(),
     requiresPageReload: false,
   },
+  [profilingUseLegacyFlamegraphAPI]: {
+    category: [observabilityFeatureId],
+    name: i18n.translate('xpack.observability.profilingUseLegacyFlamegraphAPI', {
+      defaultMessage: 'Use profiling legacy Flamegraph API',
```

Comment:
```suggestion
      defaultMessage: 'Use legacy Flamegraph API in Universal Profiling',
```

Minor suggestion for clarity.

Do we also need a longer description? I guess it depends on if we expect this toggle to exist after the 8.11 release.

Reply:
The idea is that the setting will go away after 8.11.